metadata-jobs/mce-consumer-job/build.gradle

```diff
@@ -1,17 +1,73 @@
 plugins {
     id 'org.springframework.boot'
     id 'java'
+    id 'com.palantir.docker'
+}
+
+apply from: "../../gradle/versioning/versioning.gradle"
+
+ext {
+    docker_registry = 'linkedin'
+    docker_repo = 'datahub-mce-consumer'
 }
 
 dependencies {
-    compile project(':metadata-jobs:mce-consumer')
-    compile(externalDependency.springBootStarterWeb) {
+    implementation project(':metadata-service:factories')
+    implementation project(':metadata-service:auth-filter')
+    implementation project(':metadata-jobs:mce-consumer')
+    implementation project(':entity-registry')
+
+    implementation(externalDependency.springBootStarterWeb) {
         exclude module: "spring-boot-starter-tomcat"
     }
-    compile externalDependency.springBootStarterJetty
-    compile externalDependency.springKafka
+    implementation externalDependency.springBootStarterJetty
+    implementation externalDependency.springKafka
+    implementation spec.product.pegasus.restliDocgen
+    implementation spec.product.pegasus.restliSpringBridge
+    implementation externalDependency.slf4jApi
+    implementation externalDependency.log4j2Api
+    compileOnly externalDependency.lombok
+    implementation externalDependency.logbackClassic
+
+    runtime externalDependency.mariadbConnector
+    runtime externalDependency.mysqlConnector
+    runtime externalDependency.postgresql
+
+    annotationProcessor externalDependency.lombok
+
+    testImplementation externalDependency.springBootTest
+    testCompile externalDependency.mockito
+    testCompile externalDependency.testng
 }
 
 bootJar {
     mainClassName = 'com.linkedin.metadata.kafka.MceConsumerApplication'
-}
\ No newline at end of file
+    requiresUnpack '**/restli-servlet-impl.jar'
```

Comment:
Might be useful to add a comment about why unpacking is required here for future readers